### PR TITLE
Add site header

### PR DESCRIPTION
**Pull request by iHiD**
Screenshot 2020-10-30 at 16 15 44

This doesn't quite feel right, and needs some work around positioning, but it'll do for Halloween :)

### CODE REVIEW

File: app/views/layouts/_header.html.haml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+%header#site-header
+  .lg-container.container
+    = link_to root_path, class: "exercism-link" do
+      = icon "exercism-with-logo-black", "Exercism"
+
+    %ul.primary-links
```
Comment (taiyab) [inline app/views/layouts/_header.html.haml]: Is there a reason not to use a `<nav>` tag here?

**Review by taiyab — APPROVED** [review]

File: app/views/layouts/_header.html.haml (hunk 1 above)
Comment (iHiD) [reply app/views/layouts/_header.html.haml]: Nope. Added it 🙂

File: app/views/layouts/_header.html.haml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      = icon "exercism-with-logo-black", "Exercism"
+
+    %nav
+      %ul
+        %li
+          = link_to "#" do
+            = graphical_icon(:dashboard)
+            Dashboard
+
+        %li.selected
```
Comment (SleeplessByte) [inline app/views/layouts/_header.html.haml]: You'll want to add: `aria-current="page"`.

Comment (iHiD) [reply app/views/layouts/_header.html.haml]: Done. Thanks :)

**Pull request merged**